BZ2229164: Minor update to bridge NAD YAML (#63552)
openshift/openshift-docs: modules/virt-creating-linux-bridge-nad-cli.adoc

    "type": cnv-bridge, <4>
    "bridge": bridge-interface, <5>
    "macspoofchk": true, <6>

>     "vlan": 1, <7>

Reviewer: We should make it something else than 1 (which is typically the default VLAN)

<5> The name of the Linux bridge configured on the node.
<6> Optional: Flag to enable MAC spoof check. When set to `true`, you cannot change the MAC address of the pod or guest interface. This attribute provides security against a MAC spoofing attack by allowing only a single MAC address to exit the pod.
<7> Optional: The VLAN tag. No additional VLAN configuration is required on the node network configuration policy.

> <8> Indicates whether the default VLAN must be preserved on the virtual ethernet (veth) end connected to the bridge. This must be set to `false`.

Reviewer: ```suggestion
<8> Indicates whether the VM will be connected to the bridge through the default VLAN.
```
